### PR TITLE
#206 Changed minimal count

### PR DESCRIPTION
**Pull request by PawelMelnik**
- Minimal count of columns chaged from 5 to 15

### CODE REVIEW

File: app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java [hunk 1]
```diff
@@ -372,14 +372,15 @@ private void setSizes() {
         _isScreenWidth = isScreenWidth();
         computeSizes();
         int bmWidth = _bmpImageColumns.getWidth();
-        if (bmWidth < 90) {
+        if (bmWidth < EditImageActivity.MAX_SIZE) {
             _sbColumns.setMax(bmWidth - EditImageActivity.MIN_SIZE);
         } else {
-            _sbColumns.setMax(90 - EditImageActivity.MIN_SIZE);
+            _sbColumns.setMax(
+                    EditImageActivity.MAX_SIZE - EditImageActivity.MIN_SIZE);
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java]: Check coding style

File: app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java [hunk 2]
```diff
@@ -325,7 +335,7 @@ public void onResume() {
         if (_columns == 0) {
             int bmWidth = _bmpCurrentImage.getWidth();
             int bmHeight = _bmpCurrentImage.getHeight();
-            _columns = bmWidth < 90 ? bmWidth : 45;
+            _columns = bmWidth < MAX_SIZE ? bmWidth : 45;
```
Comment (MaximSmolskiy) [inline app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java]: Probably, it worth to replace "45" by constant (DEFAULT_SIZE, for example), how it was done with "15" and "90".

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Review by maik-nack — APPROVED** [review]
Good job!

**Pull request merged**